test/parser/expression/Call.test.js

```diff
@@ -25,10 +25,45 @@ describe("parser call expressions", () => {
         expect(statements).toMatchSnapshot();
     });
 
+    it('does not invalidate the rest of the file on incomplete statement', () => {
+        const { tokens } = brs.lexer.Lexer.scan(`
+            sub DoThingOne()
+                DoThin
+            end sub
+            sub DoThingTwo()
+            end sub 
+        `);
+        const { statements, errors } = parser.parse(tokens);
+        expect(errors.length).toBeGreaterThan(0);
+
+        //ALL of the errors should be on the `DoThin` line
+        let lineNumbers = errors.map(x => x.location.start.line);
+        for (let lineNumber of lineNumbers) {
+            expect(lineNumber).toEqual(3);
+        }
+        expect(statements).toMatchSnapshot();
+    });
+
+    it('does not invalidate the next statement on a multi-statement line', () => {
+        const { tokens } = brs.lexer.Lexer.scan(`
+            sub DoThingOne()
```

Comment:
Like in #215, I typically try to avoid making these tests depend on the lexer's implementation.  I think I'm okay with leaving this as-is because of how verbose this test would be otherwise (since locations _are_ in fact important here).